service/tracker/execution.go

```diff
@@ -61,7 +61,7 @@ func NewExecution(log zerolog.Logger, db *badger.DB, stream RecordStreamer) (*Ex
 		return nil, fmt.Errorf("could not retrieve root header: %w", err)
 	}
 	var sealID flow.Identifier
-	err = db.View(operation.LookupBlockSeal(blockID, &sealID))
+	err = db.View(operation.LookupBySealedBlockID(blockID, &sealID))
```

Comment:
this should be `LookupLatestSealAtBlock`. See https://github.com/onflow/flow-go/pull/2551 for context